src/coroutine.rs

```diff
@@ -168,6 +168,14 @@ impl<Input, Yield, Return, Stack: stack::Stack> Coroutine<Input, Yield, Return,
         Input: 'static,
         Yield: 'static,
         Return: 'static,
+    {
+        unsafe { Self::with_stack_unchecked(stack, f) }
+    }
+
+    /// Unsafe version of [`Coroutine::with_stack`], i.e. without lifetime bounds.
```

Comment:
This needs a `# Safety` comment explaining what safety invariants must be manually maintained.